action.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: 'Docker-build-release'
+description: 'Build Docker image and push to GHCR'
+inputs:
+  organization:
+    description: 'Organizational host for the image. Defaults to the organization of the calling repository.'
```

Comment:
Is it not possible to include that specific default value here?

Reply:
That... is a great question. I'm already doing it with `${{ github.token }}`, so it feels like `${{ github.event.organization.login }}` should work as well...

Reply:
It totally works - see https://github.com/uclahs-cds/docker-internal-tests/actions/runs/10427550837/job/28882459276.

Fixed in 280749743225d07d1325bf1b85ff201d3a0bef30.